src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.cs

```diff
@@ -282,4 +282,12 @@ protected override void Dispose(bool disposing)
         _internalDriver = null!;
         base.Dispose(disposing);
     }
+
+    /// <inheritdoc />
+    public override ComponentInformation QueryComponentInformation()
+    {
+        var ret = new ComponentInformation(this, "Generic Raspberry Pi Wrapper driver", ComponentState.Active);

```

Comment:
Not sure if the `Wrapper driver` part of the description could be better.

Reply:
Neither am I...